Antitrust-Policy.md

```diff
@@ -0,0 +1,64 @@
+* Discussions relating to any individual technology or service provider that may have the effect of excluding them from any market or influencing the business conduct of others toward such provider.
+* Discussions relating to actual or potential customers that might have the effect of influencing the business conduct of others toward such customers.
+* Discussions relating to bids or intentions to bid for particular products or services including the procedures for responding to bid invitations or specific terms and conditions.
+* Discussions relating to individual company market shares for any product or service.
+
+In connection with all CNOE-related meetings, conference calls, repositories, or other forums for exchange of information, Members and Participants agree to:
+
+* Use best efforts to prepare agendas in advance and adhere to prepared agendas.
+* Use best efforts to document meeting minutes to accurately reflect what transpired and distribute or make available the same to all participants.
+* Object to any discussions that appear to violate this Policy or Antitrust Laws, discontinue participation in any meeting or call if such discussions continue, insist that such objection protest be noted in the meeting minutes, and make such objection protest known to the Legal Counsel of all CNOE core members.
+
+## Development of Recommended Best Practices and Reference Implementations
+CNOE shall discuss and cultivate the Recommended Best Practices and Reference Implmentations consistent with this Policy, CNOE’s Guiding Principles, CNOE’s Mission Statement and not for any anti-competitive purpose or effect.  CNOE working groups shall develop and select Recommended Best Practices based on their quality, technological attributes and ability to address the needs of developer platform users. 
+
+Recommended Best Practices shall not incorporate confidential or proprietary information or any material that infringes or misappropriates any third-party intellectual property rights. Recommended Best Practices shall not require a provider, developer, or end-user of Recommended Best Practice obtain a license or permission from any third party to use such technology except for any open source software license pursuant to which the Recommended Best Practice source code is licensed.
+
+CNOE and its working groups shall make available all draft Recommended Best Practices on the CNOE website or public repositories for a reasonable period of time prior to making formal recommendations.  CNOE and its Members welcome input from Participants and non-members and shall consider such feedback prior to making formal recommendations.  As set forth above, participation in CNOE is open to all participants consistent with CNOE’s governing charter. This expressly includes those who may be affected by a Recommended Best Practice recommended by CNOE who shall be given the opportunity to comment on the Recommended Best Practice.  
```

Comment:
I have concerns about the operational burden that this places on the project.

Reply:
"CNOE and its working groups shall make available all draft Recommended Best Practices on the CNOE website or public repositories for a reasonable period of time prior to making formal recommendations. "

How would this manifest in our operations?  We would need to publish technology choices for review before we made a decision?

" CNOE and its Members welcome input from Participants and non-members and shall consider such feedback prior to making formal recommendations."

Again, what kind of operation burden does this place on the project?  How would this manifest in practice?

"This expressly includes those who may be affected by a Recommended Best Practice recommended by CNOE who shall be given the opportunity to comment on the Recommended Best Practice."

Does this ability to comment include a requirements to publish dissent on decisions along with our guidance?

Reply:
I would also like to understand the reasoning behind making the draft available first before making it formal.

Reply:
Giving it another read, I believe making the recommended best practices into PRs that are available for the general public to potentially comment on, should satisfy this last paragraph. I dont think this snippet requires us to actively search for them or try to satisfy them.

Also as indicated already, participation in CNOE should remai `consistent with CNOE’s governing charter` which helps us navigating the decision making process